frontend/src/metabase/home/components/EmbedHomepage/EmbedHomepageView.tsx

```diff
@@ -57,14 +61,14 @@ export const EmbedHomepageView = (props: EmbedHomepageViewProps) => {
           <Text color="text-light" size="sm">
             {/* eslint-disable-next-line no-literal-metabase-strings -- only visible to admins */}
             {jt`Because you expressed interest in embedding Metabase, we took this step for you so that you can more easily try it out. You can turn it off anytime in ${(
-              <Link
-                to="admin/settings/embedding-in-other-applications"
+              <Anchor
```

Comment:
Changed from `Link > Anchor` to `<Anchor component={Link}` to address warning about nested `a` tags